src/balatro_gym/interfaces.py

```diff
@@ -40,8 +40,14 @@ class Tarot(Card):
     pass
 
 
-class BoosterPack:
-    pass
+@runtime_checkable
+class Booster(Protocol):
+    cash_value: int
```

Comment:
Nit: `base_cost` and `cost` are used elsewhere to describe cost to buy, e.g. in `JokerBase`. We should unify for consistency. I have a slight preference for "cost" rather than "value" because "cost" is more explicitly descriptive of purchasing. And Boosters can only be bought